use-cases/NIC-passthrough-and-Kata.md

```diff
@@ -0,0 +1,217 @@
+# How To Pass a Physical NIC Into a Kata Container
+
+* [Before you start (Restrictions, Requirements and Assumptions)](#before-you-start-restrictions-requirements-and-assumptions)
+* [Pass a physical NIC into a Kata Container](#Pass-a-physical-NIC-into-a-Kata-Container)
+  * [Part 1 – The Host](#part-1--the-host)
+  * [Part 2 – The Container](#part-2--the-container)
+* [Test Your Setup](#Test-Your-Setup)
+
+![](./images/NIC%20passthrough%20Diagram.png)
+
+
+In this guide we walk through the process of passing a physical NIC into Kata Containers. The "usual" way that a Kata Container is wired for communication is through a bridge and virtual NIC, as can be seen in the image above (marked as default).
```

Comment:
Nit: It would help if you folded long lines somewhere between 70-80 characters throughout the document. That makes diffs easier to read.